.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+	"connector": {
+	  "name": "local",
+	  "options": {
+		"pattern": ["**", "!.git/**", "!node_modules/**"]
+	  }
+	},
+	"extends": ["development"],
+	"formatters": ["stylish"],
+	"hints": [
+	  "button-type",
+	  "disown-opener",
+	  "html-checker",
+	  "meta-charset-utf-8",
+	  "meta-viewport",
+	  "no-inline-styles:error"
+	]
+  }
```

Comment:
- [ ] As required by the previous reviewer, Kindly make sure you correctly set up your linters. Once they are correctly set up and all passed you should have **two checked buttons in green color** at the bottom of your pull request. In your linters configuration, you are missing the following folders **[.github/workflows/](https://github.com/microverseinc/linters-config/tree/master/html-css/.github/workflows)** and this **[linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml)** file with its content inside the `.github/workflows/`. You take a look at the [linters template](https://github.com/microverseinc/linters-config/tree/master/html-css) and how they are created and do the same in your root directory.
- **Also, remember to fix the **conflicts** you have with your default branch because even if you correctly set up linters you should have the **conflicts** issue.**

## Notice:
**Make sure you use this same pull request for your next review.** That means rou will not create a new one or even close it before it is approved. 

Happy coding :clap: :clap: :tada:

Reply:
Hello Tresor, thanks for the feedback. To fix the conflicts do I need to go to my code editor and push commit the changes or is there a way to change it here? And the same question for those linters folders, do I need to open a new pull request after that? Thank you again!

Reply:
Hi @alexmedinasf,
- For resolving conflicts, you can either fix them from here or even locally. But if you don't know to fix conflicts I can suggest you watch **[this video](https://www.youtube.com/watch?v=JtIX3HJKwfo)** that can help you to do so.
- And for linters configuration, you don't need to create a new pull request. Just add the changes in your existing code and then commit and push them at the same pull request. As you should use this same pull request to request a new review.

Reply:
Sorry to bother you again, @tresorsawasawa , but isn't this linter folder the first one that shows on my repository? I created the two folders and added the code to the linters file, what did I do wrong?

Reply:
Ok solved the linter and the conflict, is there any way I can solve the linter error just like I solved the conflict or do I need to go to my code editor to do it?

Reply:
Hi,
I am really sorry because I am not at my working place right now. I don't have my computer with me. 
But you can solve linters from your IDE.

Reply:
@tresorsawasawa I think i fixed everything, what do i have to do now? Again, thank you for all the help